content/6_Advanced/LCT.mdx

```diff
@@ -399,6 +657,224 @@ int main() {
 	/>
 </Resources>
 
+### Implementation
+
+We can also maintain information about subtrees by keeping track of values for the virtual subtrees of a node. When querying for information such as the subtree sum, we call $\textup{access}()$ on the node so that all of its children in the represented tree are part of virtual subtrees and then retrieve the desired value.
```

Comment:
uh i think we use texttt instead of textup

Reply:
Fixed